components/ModalDelete.js

```diff
@@ -0,0 +1,32 @@
+import styled from "styled-components";
+
+export default function ModalDelete({ onConfirm, onCancel }) {
+  return (
+    <StyledBackground>
+      <StyledContainer>
+        <p>Are you sure you want to delete this project?</p>
+        <button onClick={onConfirm}>Yes</button>
+        <button onClick={onCancel}>No</button>
```

Comment:
It's a good practice to use **type** attribute for buttons to distinguish their behaviour 🙂
```suggestion
        <button type="button" onClick={onConfirm}>Yes</button>
        <button type="button" onClick={onCancel}>No</button>
```